framework/configuration/entities-relations.rst

```diff
@@ -0,0 +1,559 @@
+     <![CDATA[<b>[@loadTextResource path="entities.Address.address_nr"/]: ${baseData.address_nr?c}</b>
+       <table style="border: none; text-align: left; width: 100%;" border="0">
+         <tbody>
+           <tr>
+             <td align="left" valign="top">[@loadTextResource path="entities.Address.tooltip.type"/]:</td>
+             <td align="left" valign="top">${baseData.relAddress_type.label}</td>
+           </tr>
+         </tbody>
+       </table>
+     ]]>
+   </display>
+
+* **language** (*javascript*, freemarker)
+   Defines which language is used to process the expression. Only freemarker should be used. ``javascript`` is **deprecated**.
+
+* **type** (string)
+   Optional. If this is not set, the display will be used as default display. If an entity needs more than one display, they can be distinguished using the type attribute. Common types are ``tooltip``, ``search`` (Used for fulltext search results) and ``resourceCalendarTooltip`` (used in Ressource-Calendar).
+
+Relations
+---------
+The relation configuration consists of a `Source- & Target-Elements`_ configuration, the `Cardinality`_ and additional `Optional Configuration`_. This configuraiton is written in and must be stored in a file using the naming convention ``Entity_name_relRelation_target.xml`` in the ``relations`` subfolder of the model folder.
+
+.. tip::
+
+   In most cases the ``Relation_target`` is the entity name of the target entity.
+
+Example-Path in an optional module: ``optional/modulename/module/model/relations/Entity_name_relRelation_target.xml``
+
+Example-Path in a customer module: ``customer/customername/share/model/relations/Entity_name_relRelation_target.xml``
+
+
+.. code-block:: xml
+
+   <?xml version="1.0" encoding="UTF-8"?>
+   <relation xmlns="http://nice2.tocco.ch/schema/relation.xsd">
+     <source entity-model="User">
+       <delete cascade="no"/>
+       <display show="false"/>
+     </source>
+     <target entity-model="User_code1"/>
+     <cardinality>n:n</cardinality>
+   </relation>
+
+Source- & Target-Elements
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: xml
+
+   <source entity-model="Membership" name="relSingle_user">
+     <delete cascade="no"/>
+     <display show="false"/>
+   </source>
+   <target entity-model="User" name="relSingle_user_membership" label="">
+     <delete cascade="no"/>
+     <display show="true" order="30"/>
+   </target>
+   <cardinality>n:0..1</cardinality>
+
+Source- and Target-Elements define how the relation works. The source as well as the target element consists of attributes and child elements.
+
+**Attributes:**
+
+* entity-model
+   The entity-model-name this name refers to the names used in the entity-configurations.
+
+* name
+   The relation-name. If not specified, it will be ``relEntity_name``. The source/target-entity and the name of the relation identifies it. If you specify a relation on two or more locations, the configurations-system will merge it to a single relation.
```

Comment:
Maybe an example when it makes sense to set a name manually? E.g. Two relations from and to the same entity.